controllers/AuthController.js

```diff
@@ -0,0 +1,26 @@
+import sha1 from 'sha1';
+import uuid4 from 'uuid';
+import dbClient from '../utils/db';
+import redisClient from '../utils/redis';
+
+class AuthController {
+  // AuthController.getConnect
+  static async getConnect(req, res) {
+    const authHeader = req.headers.authorization;
+    // The atob is for splitting the base64 encoding and then converting it to a string
+    // eslint-disable-next-line no-undef
+    const [email, password] = atob(authHeader).split(':');
+
+    const user = await dbClient.users.findOne({ email });
+    if (!user) {
+      res.status(401).json({ error: 'Unauthorized' });
+    } else if (user && sha1(user.password) === password) {
+      const userUUID = uuid4();
```

Comment:
userId would have been a good variable name imo

Reply:
Thats good, i didnt know of a better name to use because i didnt want to have a conflit with the id from the user obj from the class or better still i would have generated as a passed param to the dbClient.set method

Reply:
Thats good, i didnt know of a better name to use because i didnt want to have a conflit with the id from the user obj from the class or better still i would have generated as a passed param to the dbClient.set method